app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -27,10 +27,21 @@ class ExportWorkflowsDialog extends React.Component {
       return;
     }
 
-    apiClient.type('workflows')
-      .get(project.links.workflows)
+    const order = (project.configuration && project.configuration.workflow_order) || [];
+
+    // TODO: this API call duplicates information fetched to draw the lab sidebar.
+    // when we do the lab refactor, this should be cached somewhere
+    project.get('workflows', { fields: 'display_name' })
       .then((workflows) => {
-        this.setState({ workflows });
+        let result = workflows;
+
+        if (order) {
```

Comment:
`project.configuration.workflow_order` is an array and an empty array checked this way will be truthy. When `order` is an empty array, the `result` variable will set as an empty array and the dialog will not have workflows to select from.